recipes/dask_traj/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dask_traj" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "165f98016f59944cbd5960306ab26b69957f0abb14f8d4496d0dbe31977e59ad" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
```

Comment:
You probably just need pip here, not setuptools

Reply:
Good catch, thank you